ql/src/java/org/apache/hadoop/hive/ql/ddl/table/misc/rename/AbstractAlterTableRenameAnalyzer.java

```diff
@@ -48,7 +54,12 @@ protected void analyzeCommand(TableName tableName, Map<String, String> partition
     if (AcidUtils.isTransactionalTable(table)) {
       setAcidDdlDesc(desc);
     }
-    addInputsOutputsAlterTable(tableName, null, desc, desc.getType(), false);
```

Comment:
I think we are throwing out a lot of logic by not calling this method - and that might be the cause of the sheer volume of failures/test output changes

Reply:
I tried to implement essential logic in that method here which is organizing input and output object lists. But I now added back the method and added the output list in a way that we would need it in Apache Ranger.